cmd/asset-syncer/utils_test.go

```diff
@@ -619,7 +636,9 @@ func Test_fetchAndImportFiles(t *testing.T) {
 		// don't return an error when checking if files already exists
 		m.On("One", mock.Anything).Return(nil)
 		dbSession := mockstore.NewMockSession(&m)
-		manager := &mongodbAssetManager{mongoConfig: datastore.Config{}, dbSession: dbSession}
+		man := dbutils.NewMongoDBManager(datastore.Config{})
+		man.DBSession = dbSession
+		manager := &mongodbAssetManager{man}
```

Comment:
With 10 occurrences of the same 3 lines, I reckon it'd be worth defining small helper function `getManagerForEmptyConfig()` or similar.